### PR TITLE
Feat : Added auth pages

### PR DESCRIPTION
**Pull request by sushil-mod**
# Added auth pages 

- Login page 
- Signup psge

## what to check 

- Login.jsx ;- src/pages/auth/Login.jsx
- Signup :- src/pages/auth/Signup.jsx

### screenshot 
login
sign

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
It's looks fine

File: src/pages/auth/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+   <div className="flex-center height-vh-100">
+        <form className="form-auth flex-center flex-col bx-shadow">
+            <div className="form-logo wd-100">
+                <Link to="/"> 
+                    <div className="nav-logo flex-center flex-col">
+                        <i className="fas fa-jedi nav-logo-icon"></i>
+                        <span className="nav-logo-txt">Jedi Store</span>
+                    </div>
+                </Link>
+            </div>
```
Comment (mariyasada) [inline src/pages/auth/Signup.jsx]: it's jedi note it or jedi store?

**Review by mariyasada — APPROVED** [review]
good to merge

File: src/pages/auth/Signup.jsx (hunk 1 above)
Comment (sushil-mod) [reply src/pages/auth/Signup.jsx]: ;-) forgot to rename as its taken from ecom , thank you for pointing out .

**Pull request merged**